docker-entrypoint.sh

```diff
@@ -21,16 +22,22 @@ OPENNMS_OVERLAY_JETTY_WEBINF=/opt/opennms-jetty-webinf-overlay
 OPENNMS_UPGRADE_GUARD=${OPENNMS_HOME}/etc/do-upgrade
 OPENNMS_CONFIGURED_GUARD=${OPENNMS_HOME}/etc/configured
 
-OPENNMS_KARAF_TPL=/root/org.apache.karaf.shell.cfg.tpl
+OPENNMS_KARAF_TPL=/opt/opennms/assets/org.apache.karaf.shell.cfg.tpl
 OPENNMS_KARAF_CFG=${OPENNMS_HOME}/etc/org.apache.karaf.shell.cfg
 
-OPENNMS_NEWTS_TPL=/root/newts.properties.tpl
+OPENNMS_NEWTS_TPL=/opt/opennms/assets/newts.properties.tpl
 OPENNMS_NEWTS_PROPERTIES=${OPENNMS_HOME}/etc/opennms.properties.d/newts.properties
 
 # Error codes
 E_ILLEGAL_ARGS=126
 E_INIT_CONFIG=127
 
+# To avoid issues with OpenShift
+umask 002
+sed -i -r "/^myuser/s/=.*/=\$RUNAS/" $OPENNMS_HOME/bin/install
+sed -i -r "/^myuser/s/=.*/=\$RUNAS/" $OPENNMS_HOME/bin/upgrade
+sed -i -r "/^myuser/s/=.*/=\$RUNAS/" $OPENNMS_HOME/bin/opennms
```

Comment:
Just to be picky :) `${OPENNMS_HOME}` vs `$OPENNMS_HOME`

Reply:
No problem ;)